hw2/05-form.html

```diff
@@ -1,20 +1,96 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
     <title>05 - Form</title>
     <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
-  </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
-    </main>
-  </body>
+</head>
+<body class="bg-dark p-2 pt-5">
+<main class="card mx-auto col-lg-4 col-md-6 col-sm-12 p-2">
```

Comment:
for better viewing and and to allow heading to be on 1 line, i would change col-lg to 5.

Reply:
I do not understand. The heading is not on line 1? Are you suggesting I increase the width of the form for better viewing on the large screen? If so, I felt form form pretty much readable, so kept it as lg-4.